themes/cinnamon.omp.json

```
@@ -0,0 +1,61 @@
+{
+  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
+  "blocks": [
+    {
+      "type": "prompt",
+      "alignment": "left",
+      "segments": [
+        {
+          "type": "root",
+          "style": "powerline",
+          "powerline_symbol": "\uE0B0",
+          "foreground": "#100e23",
+          "background": "#ffe9aa",
+          "properties": {
+            "root_icon": "\uFA90"
```

```suggestion
            "root_icon": "\uF451"
```
I meant to make this one the crown (👑) after changing back from a space theme I made. :)

---

done